src/components/NumericalInput/index.tsx

```diff
@@ -39,6 +41,13 @@ const StyledInput = styled.input<{ error?: boolean; fontSize?: string; align?: s
   }
 `
 
+function localeUsesComma(locale: SupportedLocale): boolean {
+  const n = 1.1
+  const decimalSeparator = new Intl.NumberFormat(locale).format(n)[1]
```

Comment:
What a great hack. Thanks, ECMAScript.

```suggestion
  const decimalSeparator = new Intl.NumberFormat(locale).format(1.1)[1]
```

Reply:
should we document this hack in code comments?